<launch>
    

    <param name="map_path" type="string" value="/home/vv6/Documents/lidar_map" />

    <param name="use_sc" type="bool" value="0" />

    <param name="odo_topic_name" type="string" value="/odometry" />

    <param name="imu_topic_name" type="string" value="/imu_odometry" />

    <node pkg="rtk_mapping" type="location" name="location" output="screen"/>

    <node pkg="rtk_mapping" type="result" name="result"/>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find rtk_mapping)/rviz/location2.rviz" />
    </group>

</launch>
